pages/portal/resources/create.tsx

```diff
@@ -0,0 +1,163 @@
+    const cateEle = useRef<HTMLSelectElement>(null)
+    const linkEle = useRef<HTMLInputElement>(null)
+
+    const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault()
+
+        const newResource: Resource = {
+            name: nameEle.current?.value,
+            category: cateEle.current?.value,
+            link: linkEle.current?.value,
+        }
+
+        const response = await fetch(`${urls.baseUrl as string}${urls.api.resource.add as string}`, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(newResource)
+        })
+
+        Router.push('/portal/resources')
+    }
+
+
+    return (
+        <div className="container">
+
+        <Head>
+            <title>MindVersity | Admin Portal</title>
+            <link rel="icon" href="/favicon.ico" />
+        </Head>
+
+        <Navigation />
+
+        <div className="bodyContent">
+                <h1>Create Resource</h1>
+                <div className="formContainer">
+                    <form onSubmit={handleSubmit} method="post">
+                        <label htmlFor="name">Resource Name</label>
+                        <input ref={nameEle} type="text" name="name" placeholder="Resource Name"  required/>
+                        <label htmlFor="category">Category</label>
+                        <select ref={cateEle} name="category" id="categoryChoice" >
+                            <option value="national">National</option>
+                            <option value="mindversity">Mindversity</option>
+                            <option value="help">Help</option>
+                        </select>
+                        <label htmlFor="linkornumber">Link or Phone Number</label>
+                        <input ref={linkEle} type="text" name="link" placeholder="Link or Phone Number" />
+                        <input type="submit" value="Create" className="submitInput"/>
+                    </form>
+                </div>
+            </div>
+
+            <style jsx>{`
+            .container{
+                padding-top: 50px;
+                text-align: left;
+            }
+
+            @media screen and (min-width: 1000px){
+                .bodyContent{
+                    width: auto;
+                    height: auto;
+                    position: relative;
+                    display: body;
+                    margin-left: 375px;
+                }
+            }
+
+            h1{
+                color: black;
+                padding: 0px 40px;
+            }
+
+            .formContainer{
+                width: 100%;
+                height: auto;
+                position: relative;
+                display: block;
+                padding: 20px 40px;
+                text-align: center;
+            }
+
+            input[type=text], select, textarea {
+                height: auto;
+                width: 75%;
```

Comment:
I noticed that you went with 75% for the width of the fields in this form. Should we go back and change the other forms to match that width or keep the other ones at 100%?

Reply:
The other ones can stay at 100%. Since resource names and links aren't going to be too long 75% fits better, but in the case of chapter names and descriptions they can get pretty long so it's fine to leave those as is

Reply:
Ok sounds good